mule-classloader-model/src/main/java/org/mule/tools/api/classloader/AppClassLoaderModelJsonSerializer.java

```diff
@@ -67,6 +68,9 @@ public JsonElement serialize(AppClassLoaderModel classLoaderModel, Type type,
           .registerTypeAdapter(Artifact.class, new ArtifactCustomJsonSerializer())
           .create();
       JsonObject jsonObject = (JsonObject) gson.toJsonTree(classLoaderModel);
+      // Move to the bottom the additional plugin dependencies field
```

Comment:
This is just an aesthetic change right? It wont have any problems with already serialized files/

Reply:
Yes, just to make it more readable... will no affect serialization/deserialization